frontend/src/components/item-card/ItemCard.tsx

```diff
@@ -14,13 +15,32 @@ interface Item {
 
 const ItemCard: React.FC<Props> = ({ item }) => {
     const navigate = useNavigate();
+    const [quantity, setQuantity] = useState(1);
 
     function handleItemClick() {
         console.log('Item clicked', item.id);
         navigate(`/product/${item.id}`, { state: { product: item } });
 
     }
 
+    async function handleAddToCart() {
+        console.log('Add to cart clicked', quantity);
+
+        // const response = await fetch(`http://localhost:8080/api/v1/carts/products/${item.id}/quantity/${quantity}`, {
+        //     method: 'POST',
+        //     headers: {
+        //         'Content-Type': 'application/json',
+        //     },
+        // });
+
+        // const data = await response.json();
+        // console.log('Add to cart response', data);
```

Comment:
Should this be uncommented?

Reply:
I will uncomment it in the next PR after I test it.